Dockerfile.cli

```diff
@@ -0,0 +1,16 @@
+FROM php:7.0.26-cli-jessie
```

Comment:
`7.0.26` is very specific.

Reply:
It is as there is no `Dockerfile.lock`. `7.0-cli` (which defaults to `jessie`) should maintain the current behavior instead, where a new release of the code is forced to adopt a new patch version of the `php` package.

We are going to end up having a elife php image extracted from here that we can version and update periodically, so it should not be necessary to lock the patch version here.

Reply:
:+1: to an eLife image.

Reply:
Needs to be extracted from here after this container has been integrated into multiple environments, as we are likely to discover more problems.